core/src/main/java/hudson/Functions.java

```diff
@@ -1947,7 +1947,7 @@ public boolean hyperlinkMatchesCurrentPage(String href) throws UnsupportedEncodi
      * Gets all the {@link PageDecorator}s.
      */
     public static List<PageDecorator> getPageDecorators() {
-        // this method may be called to render start up errors, at which point Hudson doesn't exist yet. see HUDSON-3608 
+        // this method may be called to render start up errors, at which point Hudson doesn't exist yet. see JENKINS-3608
```

Comment:
```suggestion
        // this method may be called to render start up errors, at which point Jenkins doesn't exist yet. see JENKINS-3608
```

Reply:
Can be handled separately as a bulk Javadoc cleanup